packages/patrol_cli/lib/src/commands/develop.dart

```diff
@@ -327,8 +333,14 @@ class DevelopCommand extends PatrolCommand {
     switch (device.targetPlatform) {
       case TargetPlatform.android:
         appId = android.packageName;
-        action = () =>
-            _androidTestBackend.execute(android, device, interruptible: true);
+        action = () => _androidTestBackend.execute(
+              android,
+              device,
+              interruptible: true,
+              showFlutterLogs: showFlutterLogs,
+              showTestSteps: showTestSteps,
+              flavor: flutterOpts.flavor,
```

Comment:
Is that intended? Seems like unrelated with the PR code

Reply:
As above